src/server/services/govuk-notify.ts

```diff
@@ -26,7 +25,7 @@ export async function sendAuthenticationEmail(email: string, authenticationLink:
       reference: "",
     });
 
-    return (result as NotifyResult).statusText === "Created";
+    return "id" in result && result.id !== undefined;
```

Comment:
I don't _think_ this will be possible? have you printed what `result` is?

it should look something like 
```
{
  status: ..,
  statusText: ..,
  headers: ..,
  config: ..,
  data: {
    id: ..
  }
}
```

the notify client uses axios - so that's why the responses are structured that way. I think it was working before because axios converts the status code to statusText for you. 201 status code is used to describe that the request "Created" a resource 

if you don't like the idea of checking for .statusText we can always just check for .status instead? 201

Reply:
Hmmm, my assumption was that this is what is returned from the `sendEmail` method.

https://docs.notifications.service.gov.uk/node.html#send-an-email-response

I wasn't aware it was edited by axios. I was just going off what the return type was from the notify package and I didn't see statusText

<img width="739" alt="Screenshot 2023-07-24 at 15 44 36" src="https://github.com/UKForeignOffice/lists/assets/1377253/0194332b-0ac1-4976-a169-38f9e065c68c">

Maybe this needs to be changed 👆